cognite/client/_api/functions.py

```diff
@@ -122,7 +122,7 @@ def create(
             env_vars (Dict[str, str]):               Environment variables as key/value pairs. Keys can contain only letters, numbers or the underscore character. You can create at most 100 environment variables.
             cpu (Number, optional):                  Number of CPU cores per function. Allowed values are in the range [0.1, 0.6], and None translates to the API default which is 0.25 in GCP. The argument is unavailable in Azure.
             memory (Number, optional):               Memory per function measured in GB. Allowed values are in the range [0.1, 2.5], and None translates to the API default which is 1 GB in GCP. The argument is unavailable in Azure.
-            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", `None`], and `None` translates to the API default which currently is "py38". The runtime "py38" resolves to the latest version of the Python 3.8 series.
+            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", "py310", `None`], and `None` translates to the API default which currently is "py38". The runtime "py38" resolves to the latest version of the Python 3.8 series.
```

Comment:
Just a suggestion:
```suggestion
            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", "py310", `None`], and `None` translates to the API default which will change over time. The runtime "py38" resolves to the latest version of the Python 3.8 series.
```
I also think you should update the official API docs before doing the Python SDK:

<img width="1146" alt="image" src="https://github.com/cognitedata/cognite-sdk-python/assets/8521241/91945c02-91f4-4d91-9f17-5ff319522cc6">

Reply:
Thanks!

I've updated it here: https://github.com/cognitedata/doc.cognitedata.com/pull/1697

Reply:
Ah, the API docs.
I'll fix before merging.

Reply:
https://github.com/cognitedata/service-contracts/pull/2201

Reply:
# 🚀🚀